### DISCUSSION TITLE
Agenda for the Flatcar community call on Tuesday, 8th of March 5:30pm CET / 10pm IST / 4:30pm GMT / 11:30am EST / 8:30am PST

### DISCUSSION THREAD

**Discussion by miao0miao** [flatcar/Flatcar#640] [Flatcar Office Hours]
# Agenda for the Flatcar community call on Tuesday,  8th of March 5:30pm CET / 10pm IST / 4:30pm GMT / 11:30am EST / 8:30am PST

- Github issue: https://github.com/flatcar-linux/Flatcar/issues/641
- Zoom link: [https://us06web.zoom.us/j/84336116610](https://us06web.zoom.us/j/84336116610)
  - Meeting ID: 843 3611 6610
  - Passcode: 444888
- Youtube live stream / recording: https://www.youtube.com/watch?v=SqTv5pGLID0
- Community Calls calendar (all future calls): [link](https://calendar.google.com/calendar/u/0/embed?src=c_ii991mqrpta9en8o7ofd4v19g4@group.calendar.google.com)
  - iCal version: [link](https://calendar.google.com/calendar/ical/c_ii991mqrpta9en8o7ofd4v19g4%40group.calendar.google.com/public/basic.ics)
- other ways to keep in touch [here](https://github.com/flatcar-linux/Flatcar#participate-and-contribute)

## News
* Alpha 3165.0.0, Beta 3139.1.0, Stable 3033.2.3, LTS 2605.26.1 release this week
  - CVE-2022-0847 ["DirtyPipe"](https://dirtypipe.cm4all.com/) is patched in all channels :+1: 
* Notable changes:
  - cgroupv1 can be selected during provisioning - create `/etc/flatcar-cgroupv1` through ignition
  - Cilium networking bug fixes (kernel regression, systemd conflict)
  - kerberos support in NFS utils in Alpha
 * Hosted a "Doc Writing" day on Friday 4th March 2022
  - 11 docs PRs opened, 10 of these already merged

## Spotlight
Roadmap items:
* sysext @pothos 
* portage-stable sync with gentoo @krnowak 
* release pipeline automation @sayanchowdhury 
* testing/QA infrastructure @tormath1 
* TPM security @jepio 

## Q&A

**Comment by pothos** [comment]
Here are some systemd-sysext roadmap items we can mention:
Using sysext images for OEM tools and updates: https://github.com/flatcar-linux/Flatcar/issues/60
Using sysext images as Torcx replacement: https://github.com/flatcar-linux/Flatcar/issues/443
Larger roadmap item for sysext image usage to extend the base OS where containers fall short: https://github.com/flatcar-linux/Flatcar/issues/548

**Comment by krnowak** [comment]
The portage-stable as gentoo snapshot roadmap items would be:

- update packages and eclasses in portage-stable (doing it now in yearly batches) (#645)
- simplify installation of configuration to /etc and handle configuration updates on OS updates (#647)
  - end goal 1: packages can install their stuff to /etc, so fewer ebuilds are modified by us and can go back to portage-stable
  - end goal 2: contents of /etc can be updated on OS update by doing 3way merge between old OS, new OS and current /etc.
- update coreos-overlay packages and automate their updates if possible (#648)
- prepare the ground for switching from portage-stable to gentoo snapshots (#649)

**Comment by sayanchowdhury** [comment]
I would like to cover the roadmap item (https://github.com/flatcar-linux/Flatcar/issues/632) and the associated (https://github.com/flatcar-linux/Flatcar/issues/650)

**Comment by tormath1** [comment]
We could present this roadmap item: https://github.com/flatcar-linux/Flatcar/issues/652 - a focus on the Flatcar's tests which includes the following tasks:
* https://github.com/flatcar-linux/Flatcar/issues/655
* https://github.com/flatcar-linux/Flatcar/issues/653
* https://github.com/flatcar-linux/Flatcar/issues/651
* https://github.com/flatcar-linux/Flatcar/issues/654

**Comment by jepio** [comment]
I'll briefly talk about our strategy to enhance Flatcar security through TPM (https://github.com/flatcar-linux/Flatcar/issues/630)
